### PR TITLE
Rholang: Refactor common assertions to methods.

### PR DESCRIPTION
**Pull request by goral09**
## Overview
Refactor few common assertions in `ReduceSpec.scala` to methods.

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.
Add link to corresponding JIRA issue.

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You assigned one person to review this PR

### If you are not a member of the core development team, please confirm:
- [ ] You signed the commit. Merging requires a signature. Please see the [developer wiki](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain) for instructions.
- [ ] Your GitHub account is also an account with [Travis CI](https://travis-ci.org). Unit tests will not run on your PR unless you have an account with Travis. Merge requires passed unit tests.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala [hunk 1]
```diff
@@ -856,6 +670,8 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
         } yield space.store.toMap
         Await.result(inspectTaskReceiveFirst.runAsync, 3.seconds)
     }
+
+    checkData(receiveFirstResult)(channel, Seq(GString("Success")), mergeRand)
```
Comment (KentShikama) [inline rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: Did you forget to delete the line below this?

Comment (goral09) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: Yes, definitely. Thanks

Comment (goral09) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: Fixed.

File: rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala [hunk 2]
```diff
@@ -61,6 +62,54 @@ trait PersistentStoreTester {
 class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
   implicit val rand: Blake2b512Random = Blake2b512Random(Array.empty[Byte])
 
+  def checkData(
```
Comment (ArturGajowy) [inline rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: I'd put those on the bottom so as not to intimidate the readers from the very header of the class :)

Comment (ArturGajowy) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: Oh, I'm late to the party :D

Comment (goral09) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: you said yourself that you want to have the methods defined before they are used :)

Comment (ArturGajowy) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala]: Would approve anyway :D

**Pull request merged**